peps/pep-0740.rst

```diff
@@ -116,8 +116,8 @@ areas of Python packaging:
    metadata within the cryptographic envelope.
 
    For example, to prevent domain separation between a distribution's name and
-   its contents, this PEP proposes that digital attestations be performed over
-   ``HASH(name || HASH(contents))`` rather than just ``HASH(contents)``.
+   its contents, this PEP uses in-toto Statements to bind the distribution's
```

Comment:
Maybe link this first occurrence of "in-toto Statements" to some relevant page at the in-toto project?

(Maybe https://slsa.dev/blog/2023/05/in-toto-and-slsa or something	 similar?)

At first I was a bit confused by this Latin phrase and capitalised "Statements". Perhaps the intended audience will understand, but a link will help, and possibly even a short intro or slight rewording (e.g. 'uses "Statements" from the in-toto project').

Reply:
Thanks, that reword sounds good! Changing now.